src/Module/ModulePortfolios.php

```diff
@@ -208,12 +235,156 @@ protected function parsePortfolio(Portfolio $objItem, bool $blnAddArchive = fals
+    protected function findRemoteItems($config, $feed, $page, $limit): Collection
+    {
+        $ch = curl_init();
+        $params = $this->formatConfigForRemote($config, $feed);
+        $url = $feed->readFromRemoteUrl . '/api/portfolio/items/' . $page . '/' . $limit . '?' . $params;
+
+        curl_setopt($ch, CURLOPT_URL, $url);
+        curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
+
+        $request = curl_exec($ch);
+        curl_close($ch);
+        $data = json_decode($request, true);
+
+        // We need to format a Collection of Portfolio
+        $items = [];
+        foreach ($data as $item) {
+            $objModel = new Portfolio();
+            $objModel->pid = $feed->id;
+
+            foreach ($item as $c => $v) {
+                switch($c) {
+                    case 'category':
+                        // skip
+                    break;
+
+                    default:
+                        $objModel->{$c} = $v;
+                }
+            }
+
+            $items[] = $objModel;
+        }
+
+        $objCollection = new Collection($items, 'tl_wem_portfolio');
+
+        return $objCollection;
+    }
+
+    /**
+     * Count items from remote
+     * 
+     * @var array config
+     * @var PortfolioFeed feed
+     *
+     * @return Portfolio
+     * 
+     * @throws \Exception
+     */
+    protected function countRemoteItems($config, $feed): int
+    {
+        $ch = curl_init();
+        
+        $params = $this->formatConfigForRemote($config, $feed);
+        $url = $feed->readFromRemoteUrl . '/api/portfolio/count?' . $params;
+
+        curl_setopt($ch, CURLOPT_URL, $url);
+        curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
+
+        $request = curl_exec($ch);
+        curl_close($ch);
+        $data = json_decode($request, true);
+
+        return $data['items'];
+    }
+
+    /**
+     * Find a specific item from remote
+     * 
+     * @var mixed item
+     * @var PortfolioFeed feed
+     *
+     * @return Portfolio
+     * 
+     * @throws \Exception
+     */
+    protected function findRemoteItem($item, $feed): Portfolio
+    {
+        $ch = curl_init();
+        $params = $this->formatConfigForRemote([], $feed);
+        $url = $feed->readFromRemoteUrl . '/api/portfolio/item/' . $item . '?' . $params;
+
+        curl_setopt($ch, CURLOPT_URL, $url);
+        curl_setopt($ch, CURLOPT_FOLLOWLOCATION, true);
+        curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
+
+        $request = curl_exec($ch);
+        curl_close($ch);
+        $data = json_decode($request, true);
+
+        $objModel = new Portfolio();
+        $objModel->pid = $feed->id;
+
+        foreach ($data as $c => $v) {
+            switch($c) {
+                case 'category':
+                    // skip
+                break;
+
+                default:
+                    $objModel->{$c} = $v;
+            }
+        }
+
+        return $objModel;
+    }
+
+    protected function formatConfigForRemote($config, $feed): string
```

Comment:
Parameters typing if possible ?

Reply:
Fixed in fe448c71f043da604ab1280d6a3036f01def9948